v22.2/migration-overview.md

```diff
@@ -53,27 +50,30 @@ Consider the following CockroachDB attributes and best practices:
 
 For additional considerations specific to other databases and data formats, see the corresponding documentation linked in [Step 2. Move your data to CockroachDB](#step-2-move-your-data-to-cockroachdb).
 
-## Step 2. Move your data to CockroachDB
+### Unimplemented features
 
-We recommend [using AWS Database Migration Service (DMS) to migrate data](aws-dms.html) from any database, such as PostgreSQL, MySQL, or Oracle, to CockroachDB.
+CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and is largely compatible with PostgreSQL syntax. However, the following PostgreSQL features do not yet exist in CockroachDB:
 
-Alternatively, use `IMPORT` to [migrate CSV data](migrate-from-csv.html).
+{% include {{page.version.version}}/sql/unsupported-postgres-features.md %}
 
-You can also migrate data from the following data formats:
+If your source database uses any of the preceding features, you may need to implement workarounds in your schema design, in your [data manipulation language (DML)](sql-statements.html#data-manipulation-statements) when [moving data to the new database](#step-2-move-your-data-to-cockroachdb), or in your [application code](#step-3-test-and-update-your-application).
+
+For more details on the CockroachDB SQL implementation, see [SQL Feature Support](sql-feature-support.html).
+
+## Step 2. Move your data to CockroachDB
 
-- [Avro](migrate-from-avro.html)
-- [ESRI Shapefiles](migrate-from-shapefiles.html) (`.shp`) (using `shp2pgsql`)
-- [OpenStreetMap data files](migrate-from-openstreetmap.html) (`.pbf`) (using `osm2pgsql`)
-- [GeoPackage data files](migrate-from-geopackage.html) (`.gpkg`) (using `ogr2ogr`)
-- [GeoJSON data files](migrate-from-geojson.html) (`.geojson`) (using `ogr2ogr`)
+To migrate data from another database to CockroachDB, use the source database's tooling to extract the data to a `.sql` file. Then use one of the following methods to migrate the data:
 
-{% include {{ page.version.version }}/misc/import-perf.md %}
+- Use [AWS Database Migration Service (DMS)](aws-dms.html) to migrate data from any database, such as PostgreSQL, MySQL, or Oracle, to CockroachDB. This option is recommended for near-zero downtime migrations of large data sets, either with ongoing replication or as a one-time migration of a snapshot.
```

Comment:
nit: i think we should advertise DMS last, COPY FROM first.

Reply:
Done.